### ISSUE TITLE
Set acces token in req.session while logged in

### ISSUE THREAD

**Issue by TomasS666** [Ramon96/aria-1920#13]
**Is your feature request related to a problem? Please describe.**
At the moment the acces token is appended the hash in the url. Then in the client it's taken out and used to do requests. That's nice, we also need a way to have easy acces to the token within the server.

**Describe the solution you'd like**
I would like the acces token to be saved within the req.session object once authorisation is established. And of course to be destroyed whenever the user isn't logged in.

**Describe alternatives you've considered**
Alternative could be to fetch everything on the client. That could make it more app alike, but for a request to retrieve like data for the first view, isn't it better anyway to do that straight away from the server instead of waiting on the client-side script to inject it?

**Additional context**
Something else: this issue hold information that is valuable for the our process. It's about a strategy to conquer a challenge which relates heavily to the WAFS and PWA course. It's documentation worthy, together as a team and individually.

**Comment by TomasS666** [comment]
This is fixed, the logout issue and destroying the session should be a separate issue.
